docs/docs/rest-api/public/api/v2/types/container-status.raml

```diff
@@ -0,0 +1,56 @@
+#%RAML 1.0 DataType
+type: object
+additionalProperties: false
+properties:
+  name:
+    type: !include name.raml
+    description: name of the container specification (within the pod)
+  status:
+    type: string
+    enum: [ STAGING, STABLE, DEGRADED, TERMINAL ]
+    description: |
+      STAGING  - Container has been launched but is not yet running.
+      STABLE   - Container is running and, if health checks were specified, is healthy.
+      DEGRADED - Container is running but its health check is not passing.
```

Comment:
What would the status be if the tasks of a podInstance are Unreachable (LOST for now)? I think we need an `UNREACHABLE` here as well.